src/flyte/_internal/runtime/entrypoints.py

```diff
@@ -108,6 +111,12 @@ async def download_code_bundle(code_bundle: CodeBundle) -> CodeBundle:
 async def _download_and_load_task(
     code_bundle: CodeBundle | None, resolver: str | None = None, resolver_args: List[str] | None = None
 ) -> TaskTemplate:
+    # Adjust sys.path if necessary
```

Comment:
move this to a common function so that rusty (download) and runtime can use them